notifications_utils/clients/redis/redis_client.py

```diff
@@ -125,7 +125,7 @@ def bulk_set_hash_fields(self, mapping, pattern=None, key=None, raise_exception=
         """
         Bulk set hash fields.
         :param pattern: the pattern to match keys
-        :param mappting: the mapping of fields to set
+        :param mapptnig: the mapping of fields to set
```

Comment:
```suggestion
        :param mapping: the mapping of fields to set
```

Third time's the charm 😆

Reply:
Ahaha thanks for catching this, words are hard